install/Makefile

```diff
@@ -142,6 +143,13 @@ export HUGO_VERSION ?= 0.45.1
 hugo: hugo-$(OS)
 	@exit 0
 
+export SHFMT_VERSION ?= 2.5.1
+# Releases: https://github.com/mvdan/sh/releases
+## Install shfmt - shell parser, formatter and interpreter
+shfmt:
+	$(CURL) -o - https://github.com/mvdan/sh/releases/download/v$(SHFMT_VERSION)/shfmt_v$(SHFMT_VERSION)_$(OS)_$(ARCH) > $(INSTALL_PATH)/shfmt
```

Comment:
Use macro

Reply:
We cannot use a macro because repo name (`sh`) is not the same as the project executable. Odd convention on behalf of @mvdan.

Reply:
it's common for repos/projects to include many executables, and they can't all share the same name :)